python/pyspark/sql/functions.py

```diff
@@ -1802,10 +1802,10 @@ def from_json(col, schema, options={}):
 @since(2.1)
 def to_json(col, options={}):
     """
-    Converts a column containing a [[StructType]] into a JSON string. Throws an exception,
-    in the case of an unsupported type.
+    Converts a column containing a [[StructType]] or [[ArrayType]] into a JSON string.
+    Throws an exception, in the case of an unsupported type.
```

Comment:
nit: for python and scala here the doc wording can be confusing - it doesn't actually throw an exception right there, by itself with `to_json()`, but it might when it's trying to resolve the column, like with `select(to_json...)`